newScript.js

```diff
@@ -0,0 +1,63 @@
+var divs = new Array();
```

Comment:
Please consider using `const` or `let`.  We're in the 21st century!  You should be fine with `const` here.